coalib/misc/Exceptions.py

```diff
@@ -20,12 +17,22 @@ def get_exitcode(exception, log_printer=None):
         exitcode = exception.code
     elif isinstance(exception, VersionConflict):
         log_message = Constants.VERSION_CONFLICT_MESSAGE % str(exception.req)
-        log_printer.log_exception(log_message, exception)
+        log_exception(log_message, exception, LOG_LEVEL.ERROR)
         exitcode = 13
     elif isinstance(exception, BaseException):
-        log_printer.log_exception(Constants.CRASH_MESSAGE, exception)
+        log_exception(Constants.CRASH_MESSAGE, exception,
+                      LOG_LEVEL.ERROR)
         exitcode = 255
     else:
         exitcode = 0
 
     return exitcode
+
+
+def log_exception(message, exception, log_level=LOG_LEVEL.WARNING):
+    logging.log(log_level, message)
+    traceback_str = '\n'.join(
+        traceback.format_exception(type(exception),
+                                   exception,
+                                   exception.__traceback__))
+    logging.info('Exception was:\n' + traceback_str)
```

Comment:
fine for this PR, we should consider afterwards if we wanna do a plain `error` or `exception` log with `logging` only :+1: